### PR TITLE
set end date for memberships when archiving associated project [PEOPLE-123]

### PR DESCRIPTION
**Pull request by smnkrt**
**JIRA:** https://netguru.atlassian.net/browse/PEOPLE-123

**Changes:**
 - add Projects::EndCurrentMemberships service object
 - update ProjectsController
 - update specs

### CODE REVIEW

File: app/controllers/projects_controller.rb [hunk 1]
```diff
@@ -21,6 +21,8 @@ def show
 
   def update
     Memberships::UpdateStays.new(project.id, params[:project][:membership_ids]).call
+    Projects::EndCurrentMemberships.new(project).call if params['project']['archived'] == 'true'
```
Comment (mateuszgorniak) [inline app/controllers/projects_controller.rb]: what about using `project_params` here?

Comment (smnkrt) [inline app/controllers/projects_controller.rb]: Good idea!

Comment (mateuszgorniak) [inline app/controllers/projects_controller.rb]: the same you might use line above but it's not the part of PR :-)

Comment (smnkrt) [inline app/controllers/projects_controller.rb]: That's what i did ;)

**Comment by mateuszgorniak** [comment]
👍

**Pull request merged**